addons/website/models/website_edited.py

```diff
@@ -171,3 +171,31 @@ def _find_url(self, url, is_default_local_url=False):
                 f"'{url}?",
             ])
         return self._find(likes)
+
+    @api.model
+    def _clean(self):
+        for res_model, res_field, _ in self._get_website_edited_html_fields():
+            removed_models = []
+            if res_model in removed_models:
+                continue
+            if res_model not in self.env:
```

Comment:
Don't we have an util somewhere to get the fields related to a given module?
Imo, `_clean` should receive a list of deleted (or to be deleted) module and find those fields.
I would expect something useful in `ir.model.data` could be used?